avocado/plugins/spawners/podman.py

```diff
@@ -176,6 +184,16 @@ async def spawn_task(self, runtime_task):
 
         return returncode == 0
 
+    async def create_task_output_dir(self, runtime_task):
+        task = runtime_task.task
+        output_dir_path = os.path.join(self.job_output_dir,
+                                       task.identifier.str_filesystem)
+        output_podman_path = os.path.join('/tmp', 'avocado-results')
+
+        os.makedirs(output_dir_path, exist_ok=True)
+        runtime_task.task.setup_output_dir(output_podman_path)
+        await self._create_container_for_task(runtime_task, output_dir_path)
+
     async def wait_task(self, runtime_task):
         while True:
             if not self.is_task_alive(runtime_task):
```

Comment:
BTW, I'm doing some changes inside this file as part of the podman + eggs effort. So I guess since I'm late with that, I will have to handle the conflict at this point. :(